src/transport/primus/server.js

```diff
@@ -38,7 +38,7 @@ PrimusTransport.prototype._connect = function(url,port){
 	var address = url+':'+port;
 	if(!this._connections[address]){
 		this._connections[address] = new this._Studio.promise(function(resolve,reject){
-            var client = new _self._server.Socket(url+":"+port,_self.clientOpt);
+            var client = new _self._server.Socket('ws://' + url+":"+port,_self.clientOpt);
```

Comment:
Since this is the client-side of the connection, the user would have to know how the target service is deployed.  The environment may have some services deployed with http and others with https.  So the configuration would have to be "per target" which completely destroys the zero configuration bit.

I think we need to improve the discovery portion of our protocols to notify consumers whether to use http or https. What do you think?